kafka/kafka-2.6.0/image.yaml

```diff
@@ -30,7 +30,7 @@ packages:
   content_sets:
     x86_64:
       - rhel-7-server-rpms
-      - java-11-openjdk-devel
+      - rhel-7-server-optional-rpms
```

Comment:
This was a typo I introduced in an older commit that was preventing the builds from working! Fixing here now